script/utils/fix_imports.py

```diff
@@ -495,36 +571,259 @@ def organize_all_imports():
+                    print(f"  ❌ {issue}")
+    
+    print(f"\n📊 Summary:")
+    print(f"   Files checked: {len(files)}")
+    print(f"   Files with absolute imports: {total_files_with_issues}")
+    print(f"   Total absolute imports: {total_issues}")
+    
+    if total_files_with_issues > 0:
+        print(f"\n💡 Run with --fix-relative to convert these to relative imports")
+        return False  # Return False to indicate issues found (for CI)
+    
+    return True
+
+def fix_all_relative_imports():
+    """Convert all absolute imports to relative imports in all Solidity files"""
+    files = get_all_solidity_files()
+    print(f"Converting to relative imports in {len(files)} Solidity files...\n")
+    
+    total_files_fixed = 0
+    
+    for file_path in files:
+        if file_path.strip():
+            print(f"Processing: {file_path}")
+            if fix_relative_imports_in_file(file_path):
+                total_files_fixed += 1
+                print(f"  ✅ Converted to relative imports")
+            else:
+                print(f"  ℹ️  No absolute imports found")
+    
+    print(f"\n📊 Summary:")
+    print(f"   Files processed: {len(files)}")
+    print(f"   Files with imports converted: {total_files_fixed}")
+
 def main():
     """Main function"""
-    parser = argparse.ArgumentParser(description='Solidity import organizer and unused import detector')
+    parser = argparse.ArgumentParser(
+        description='Solidity import organizer, unused import detector, and relative path converter',
+        epilog='''
+Examples:
+  %(prog)s --check-unused              # Check for unused imports (default)
+  %(prog)s --fix-unused                # Remove unused imports
+  %(prog)s --organize                  # Organize imports by priority AND convert to relative paths
+  %(prog)s --organize --no-relative    # Organize imports by priority only (skip relative conversion)
+  %(prog)s --check-order               # Check import order (CI-friendly)
+  %(prog)s --check-relative            # Check for absolute imports (CI-friendly)
+  %(prog)s --fix-relative              # Convert absolute to relative imports only
+  %(prog)s --check-relative --file src/hub/Hub.sol  # Check specific file
+
+The --organize command now automatically converts to relative imports to ensure
+the repository can be used as a dependency without import path conflicts.
+Use --no-relative to disable this behavior if needed.
+        ''',
+        formatter_class=argparse.RawDescriptionHelpFormatter
+    )
     parser.add_argument('--check-unused', action='store_true', help='Check for unused imports')
     parser.add_argument('--fix-unused', action='store_true', help='Remove unused imports from all files')
-    parser.add_argument('--organize', action='store_true', help='Organize imports according to priority')
+    parser.add_argument('--organize', action='store_true', help='Organize imports according to priority and convert to relative paths')
```

Comment:
Informational. IIUC, the conversion from absolute -> relative paths will be supported in the tooling for later usages. We should force at the CI level that the conversion always happens. I understand that's de default behavior, so not required CI changes